### DISCUSSION TITLE
Save external data like Archives

### DISCUSSION THREAD

**Discussion by kooolarpan** [trakt/showly#511] [Ideas / Feature Requests]
If I uninstall Showly and reinstall it. The movies and series that are Archived will come to the progress section. So, it would be cool if we can save the external data like Archives which is not synced to the Trakt in the storage or maybe in Firebase.

**Comment by conradseba** [comment]
Archive should be synched with Hidden list in Trakt. I ignore why this is not the case since it seems so obvious to me.

**Comment by blackholeearth** [comment]
İ agree . 

it should create a list  "showly_archived_list" in my trakt account . 
And it should Sync the archived series with account.

**Reply by michaldrabik** [reply]
As mentioned in #534 I will be remaking Archive into "Hidden" section which will sync with designated Trakt section ("Hidden Items")
